### ISSUE TITLE
Incorrect Unirule mapping:   fap7 adenylate kinase activity (Matrix)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#4108]
**UniProt UniRule (ID and label):**

GO:0004017 | adenylate kinase activity | IEA with UR000165857

**Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPCC830.11c

**Description of issue**

Although this does have a generic adenylate kinase activity, this activity does not appear to be physiologically relevant:

see https://pubmed.ncbi.nlm.nih.gov/24823650/
and subsequent publications.

**Comment by ValWood** [comment]
also
Rhea
GO:0004017 | adenylate kinase activity | IEA with 12973 | GO_REF:0000116



[GO:0004017](https://www.pombase.org/term/GO:0004017)	
[adenylate kinase activity](https://www.pombase.org/term/GO:0004017)
[IEA](http://wiki.geneontology.org/index.php/Inferred_from_Electronic_Annotation_(IEA)) with [12973](https://www.rhea-db.org/rhea/12973)	[GO_REF:0000116](https://www.pombase.org/reference/GO_REF:0000116)

**Comment by esperetta** [comment]
@pedruzzi Could you please see if you can review this HAMAP rule for pombe (Q9UU88)? Thanks.

**Comment by ValWood** [comment]
If this causes a problem I can add a NOT annotation instead....

**Comment by esperetta** [comment]
Hi Val, sorry for the delay, yes please add NOT for the moment, while we look into how to change the entry and associated HAMAP rule.

**Comment by PedroRaposo** [comment]
Hi @pedruzzi , could we follow on this, please?

**Comment by pedruzzi** [comment]
Sure, thanks for the reminder. On it right now

**Comment by pedruzzi** [comment]
ok, after having had a look at the entries/rule and the paper, here's what I found:

- technical: changing the HAMAP rule will not prevent the GO term from being propagated. It gets the GO term from PAINT/PAN-GO, InterPro, Rhea and EC2GO, which all would need to be corrected.

- curation: I don't absolutely agree on removing the catalytic activity from FAP7/AK6. In the very paper you cite (https://pubmed.ncbi.nlm.nih.gov/24823650/) they show that free AK6 (yeast yAK6 and archaeal AK6 aAK6) have AK activity - and human AK6 has as well - and that Rps14 binding competes with substrate AMP binding. 
While AK activity may not be needed (but ATP hydrolysis is!) for ribosome biogenesis, the physiological role of AK6, AK6 is not stably associated with Rps14 and may well exert AK activity when it's free. Authors also discuss in the paper that ribosome biogenesis would be inhibited at elevated levels of AMP in the nucleus, which signals reduced energy availability, at which point AMP competes for Rps14 and AK activity may still occur.

In short, I will update the corresponding entries to reflect these findings, pointing out that the physiological function of the protein does not need AK activity, but stating the fact that the free protein still has AK activity that may compete with its function. I'm afraid this evidence is not enough to remove the Rhea reaction for this entry at this point.

ps. SGD has also annotated the GO term for AK activity with IDA based on this paper.

**Comment by ValWood** [comment]
OK, actually that makes sense if it's a chaperone it won't be permanently associated with Rps14.
A 'proper' moonlighting protein.Sorry about that one!
